src/window/building/common.h

```diff
@@ -68,6 +70,12 @@ int window_building_get_vertical_offset(building_info_context *c, int new_window
 
 void window_building_draw_employment(building_info_context *c, int y_offset);
 
+void window_building_draw_employment_without_house_cover(building_info_context *c, int y_offset);
+
+int draw_employment_info(building_info_context *c, building *b, int y_offset, int consider_house_covering);
+
+void draw_employment_details(building_info_context *c, int y_offset, int text_id, building *b);
```

Comment:
These two functions don't belong in the header since they're only used in the corresponding c file itself. You can then also skip the header.

Reply:
done